### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ruudvdent**
new pull request to update docs (preview did not work properly)

### CODE REVIEW

**Comment by ruudvdent** [comment]
@Peter9192 , please review if this can be used as temporary documentation update for a next release v3.0.0-beta.3

File: README.md [hunk 1]
```diff
@@ -5,53 +5,39 @@
...
-pip install -e .[develop]
-
-# Download sample data and update paths in config file
-https://rebrand.ly/wam2layers-example-data
-
-# Run the example backtrack script
-wam2layers backtrack floodCase_202107.yaml
-wam2layers visualize output floodCase_202107.yaml
-```
+See the [documentation](http://wam2layers.rtfd.io/) for a more detailed description.
```
Comment (Peter9192) [inline README.md:19]: I would like to keep a short overview in the GH landing page as well. Typically that's where developers end up, and it is useful for them to have a quick idea of the capabilities of the software. We can simplify it a bit though and indeed add a prominent pointer to the more extensive documentation.

File: docs/preprocessing.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This dataset adheres to the following requirements:
 
 - Data should be stored in netcdf files, one file per day.
 - Importantly, the midnight of the next day should also be included (except
-  perhaps for the last day)
+  perhaps for the last day) #TODO: @Peter, this is not correct anymore, is it?
```
Comment (Peter9192) [inline docs/preprocessing.md:33]: You're right, I can rewrite that. I'll push some commits to this branch if that's okay with you

File: docs/howtouse.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+updated code.
+
+If you do not want to do anything interactive, the core of WAM2layers - the tracking functionality - is relatively
+lightweight, and can simply be installed with pip:
+
+```
+# pip install git@github.com:WAM2layers/WAM2layers.git
+or
+pip install wam2layers
+```
```
Comment (Peter9192) [inline docs/howtouse.md:50]: This is duplicated now. 
```suggestion
```
```

File: docs/howtouse.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+TODO: merge the text below with the example?
+
+### own case
+Use the page [Case configuration](./config.md) to learn more about case configuration settings
+
+### pre-processing
+The default pre-processing works for ERA5 data for which example download scripts are provided in TODO:link to files. In case you just want to know more about what is happening during the pre-processing please refer to [Pre-processing](./prepocessing.md). 
+
+### code adaptation and contribution
+Possibly you want to adapt WAM2layers to track other climate data (e.g., output of RCMs and GCMs). In principle you can do so by only changing the Pre-processing and Case configuration without touching the core tracking module. You are encouraged to contribute your download scripts as well as prepocessing files to the WAM2layers Github repository.
```
Comment (Peter9192) [inline docs/howtouse.md:89]:
```suggestion
Possibly you want to adapt WAM2layers to track other climate data (e.g., output of RCMs and GCMs). In principle you can do so by only changing the Pre-processing and Case configuration without touching the core tracking module. You are encouraged to contribute your download scripts as well as prepocessing files to the WAM2layers Github repository so others can also benefit from your work.
```

File: docs/howtouse.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+```
+wam2layers visualize output 'config_filename'.yaml
+```
+
+TODO: explain the visualize both and snapshots options
+
+## Running WAM2layers for your own case and data
+TODO: merge the text below with the example?
+
+### own case
```
Comment (Peter9192) [inline docs/howtouse.md:82]:
```suggestion
### Own case
```

File: docs/howtouse.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+Use the page [Case configuration](./config.md) to learn more about case configuration settings
+
+### pre-processing
```
Comment (Peter9192) [inline docs/howtouse.md:85]:
```suggestion
### Pre-processing
```

File: docs/howtouse.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+The default pre-processing works for ERA5 data for which example download scripts are provided in TODO:link to files. In case you just want to know more about what is happening during the pre-processing please refer to [Pre-processing](./prepocessing.md). 
+
+### code adaptation and contribution
```
Comment (Peter9192) [inline docs/howtouse.md:88]:
```suggestion
### Code adaptation and contribution
```

**Review by Peter9192 — APPROVED** [review]
Awesome!

**Pull request merged**